articles/active-directory/manage-apps/application-proxy-configure-single-sign-on-with-ping-access.md

```diff
@@ -126,7 +126,7 @@ Finally, set up your on-premises application so that users have read access and
 4. Select **Add permissions**.
 5. In the **API permissions** page, select **Grant admin consent for \<your directory name>**.
 
-#### Collect information for the PingAccess steps
+GraphAPI#### Collect information for the PingAccess steps
```

Comment:
@mistermik @msmimart - It seems like this change introduces an error ... the H4 header formatting is removed and the #### is not visible in the line. I'm not sure what the edit is supposed to be here. Can you explain or fix? thanks.

Reply:
This just looks like something that was added by accident - @mistermik could you remove it?

Reply:
@Jak-MS can you please edit this pull request to remove that extraneous "GraphAPI" text? Then I'll get it signed off. Thanks!